gradle/libs.versions.toml

```diff
@@ -43,7 +43,7 @@ json = "20220924"
 jsonAssert = "1.5.1"
 jsonUnit = "3.2.7"
 junit = "5.10.0"
-loggingHouse = "0.2.10"
+loggingHouse = "v1.1.0"
```

Comment:
Are you sure this is the right version tag? Those usually don't have a `v` in front of them by convention but I wouldn't be surprised if there are exceptions.

Reply:
Yes, surprise:
https://github.com/truzzt/mds-ap3/packages/2045609